kubernetes/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2019
```

Comment:
Why use this image with the aspnet installation?

Could we just use the base operating system?
```suggestion
FROM mcr.microsoft.com/windows/servercore:ltsc2019
```